R/dvhydrograph-render.R

```diff
@@ -89,6 +97,11 @@ createDVHydrographPlot <- function(reportObject){
   estimated3EdgesDf <- as.data.frame(estimated3Edges, stringsAsFactors = FALSE)
   estimated4EdgesDf <- as.data.frame(estimated4Edges, stringsAsFactors = FALSE)
   comparisonEdgesDf <- as.data.frame(comparisonEdges, stringsAsFactors = FALSE)
+  
+  #remove zeros from field measurements, if present:
+  if (excludeZeroNegativeFlag && !isEmptyOrBlank(fieldVisitMeasurements)) {
+    fieldVisitMeasurements <- removeZeroNegative(fieldVisitMeasurements)
+  }
```

Comment:
if there aren't zero's, this won't do anything.
if there are zero's, but `excludeZeroNegativeFlag = FALSE`, then `logAxis` will be FALSE
if there are zero's and `excludeZeroNegativeFlag = TRUE`, then `logAxis` will be TRUE